app/views/templates/_history.html.erb

```diff
@@ -0,0 +1,25 @@
+<% unless history.empty? %>
+  <% history.each do |audit| %>
```

Comment:
It wouldn't fallback to "No history found", otherwise it would be redundant, yes.